app/derivative_services/raster_resource_derivative_service.rb

```diff
@@ -30,7 +30,7 @@ def change_set
 
   def build_cloud_file
     IngestableFile.new(file_path: temporary_display_output.path, mime_type: "image/tiff; gdal-format=GTiff", original_filename: "display_raster.tif", use: use_cloud_derivative,
-                       copy_before_ingest: false)
+                       copy_before_ingest: true)
```

Comment:
Why'd this have to change?

Reply:
Looks like when this is `true` it won't create an interstitial file, which is okay for derivatives because we just want to move it directly to the correct place, and don't care if the old file disappears (unlike for the repository.)